var/spack/repos/builtin/packages/py-fenics-instant/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyFenicsInstant(PythonPackage):
+    """Instant is a Python module that allows for instant inlining of C and C++
+    code in Python. It is a small Python module built on top of SWIG and
+    Distutils. Instant has been retired after 2017.2.0 release. It is no longer
+    needed in FEniCS and hence no longer maintained and tested. """
+
+    homepage = "https://fenicsproject.org"
+    url      = "https://bitbucket.org/fenics-project/instant/downloads/instant-2017.2.0.tar.gz"
+
+    version('2017.2.0', sha256='be24f162fd1a89b82fae002db8df0b4f111fd50db83d78c0c121015c02e45b7b')
+    version('2016.2.0', sha256='2fb4f6cf087369ca11482a945ae695066f4b9aca15d90e033f059ea1107bb874')
+    version('1.6.0',    sha256='a9c569b3fa5ce5f00465323cf53da116f4cf3d416920b5628acbdcdc15b892a7')
+
+    depends_on('py-setuptools', type="build")
+    depends_on('py-numpy', type=("build", "run"))
+    depends_on('cmake', type="run")
+    depends_on('swig', type=("build", "run"))
```

Comment:
Missing dependencies:
```python
depends_on('python@2.7:', type=('build', 'run'))
depends_on('py-six', type=('build', 'run'))
depends_on('py-subprocess32', when='^python@:2.999', type=('build', 'run'))
```
```

Reply:
ok, added dependencies and created a new PR #21775